docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -287,9 +305,9 @@ endif::[]
 +
 For example, select *test* and then click *Next*.
 
-. On the *Namespace* page, the namespace that you select depends on your {osd-name-short} environment. The namespace is the deployment space that hosts your {connectors} instances.
+. On the *Namespace* page, the namespace that you select depends on your {openshift} environment.
 +
-If you're using a trial cluster on your own {osd-name-short} environment, select the card for the namespace that was created when you added the {connectors} service to your trial cluster.
+If you're using your own {openshift} environment (ROSA or {osd-name-short} Trial), select the card for the namespace that was created when a cluster administrator added the {connectors} service to your cluster.
 +
 If you're using the hosted preview environment, click the *preview namespace* that you provisioned when you created the source connector.
 
```

Comment:
I can't tag the following lines to add my comments there directly:

1. Line 251: You might want to edit as follows:
- Original: "Select one of the following error handling policy for your ..."
- Proposed: "Select one of the following error handling polic**ies** for your ..."

2. Line 321: I can't check the UI, but just wanted to confirm that the field name is "Topic Name**s**" (plural) and not "Topic Name"?

3. Line 328: Move "Click **Next**." to a new sentence, because it's not part of selecting an error policy?